java_gen/pre-written/pom.xml

```diff
@@ -53,8 +53,8 @@
         </dependency>
         <dependency>
             <groupId>io.netty</groupId>
-            <artifactId>netty</artifactId>
-            <version>3.10.4.Final</version>
+            <artifactId>netty-all</artifactId>
+            <version>4.0.30.Final</version>
```

Comment:
`4.0.31.Final` just came out few days ago. Looks like `.31` can catch this train.